spec/linter-codeclimate-spec.js

```diff
@@ -35,4 +37,21 @@ describe('The codeclimate provider for Linter', () => {
       ),
     ),
   );
+
+  it('runs all configured engines', () =>
```

Comment:
Not really. The only difference is that if you run this spec against previous version with hardcoded engines it will fail.